lib/array_intersection.rb

```diff
@@ -1,3 +1,32 @@
 def intersection(list1, list2)
```

Comment:
This works, but it's O(n * m) in time complexity.  Using a hash you can get this to O(n + m) time complexity which is much better.  

I encourage you to think about how to do this.